### PR TITLE
V1.2.2

### PR DESCRIPTION
**Pull request by Yu-jinKim**
App updates:
 - Update region coverage app (same version v.1.0.4, different dx object)
 - Updated nirvana2vcf app (v1.1.0 → v1.1.1)

Default inputs removed:
 - Remove default target bed from Sentieon - now calls whole genome by default
 - Remove default capture bed from Picard

This change is [Reviewable](https://reviewable.io/reviews/eastgenomics/eggd_dias_single_workflow/7)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,10 +3,10 @@ DNAnexus workflow definition file of dias_single for germline analysis.
 
 -------
 
-## Current Version: 1.2.1
+## Current Version: 1.2.2
 
 ## Release Notes:
-* region_coverage from 1.0.2 to 1.0.4
+* region_coverage file id changed from applet-G0VqjYQ4PFX0XQ994vzJ0Vxg to applet-G0X0Gvj433Gzzqy90PG6vYv0
```
Comment (Addy81) [inline README.md:9]: Might be worth just mentioning that the version remained the same here.

**Review by Addy81 — COMMENTED** [review]
Might be worth just mentioning that the version remained the same here.

**Review by mattgarner — COMMENTED** [review]
> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/eastgenomics/eggd_dias_single_workflow/7)* status: 1 of 2 files reviewed, 1 unresolved discussion (waiting on @Addy81)

___
*[README.md, line 9 at r2](https://reviewable.io/reviews/eastgenomics/eggd_dias_single_workflow/7#-MYz-UN0AvtUdTc6lSVO-r2-9:-M_a1sy2BfpXopMevoux:bb6un1j) ([raw file](https://github.com/eastgenomics/eggd_dias_single_workflow/blob/77532332579be1d5f8ed363e5a2c96a2f714789f/README.md#L9)):*
<details><summary><i>Previously, Addy81 (Adriana) wrote…</i></summary><blockquote>

Might be worth just mentioning that the version remained the same here.
</blockquote></details>

Done

**Review by Addy81 — APPROVED** [review]

**Review by Addy81 — APPROVED** [review]
All looks good :)

**Pull request merged**